src/form/components/StaticField.tsx

```diff
@@ -37,31 +45,50 @@ export type Props = {
 } & (
   | {
       subtype: StaticFieldType.Button
+      title?: string
       button: ButtonProps
     }
   | {
       subtype: StaticFieldType.Text
+      title?: string
       text: string
     }
   | {
       subtype: StaticFieldType.Checklist
+      title?: string
       checklist: Array<Checklist>
     }
 )
 
 export const StaticField = (
-  props: Props & { fieldIndex: number; localeNamespace?: LocaleNamespace },
+  props: Props & { fieldIndex: number; localeNamespace?: LocaleNamespace; formReadOnly: boolean },
 ) => {
   const theme = useFormTheme()
+
   const getCSSStyle = useCSSStyles(theme, 'staticField')({})
   const getInlineStyle = useInlineStyle(theme, 'staticField')({})
   const getRowStyle = useCSSStyles(theme, 'row')({})
 
+  const appTheme = useAppTheme()
+  const getLabelStyle = useAppCSSSTyles(appTheme, 'label')({})
+
   const label = { localeNamespace: props.localeNamespace, ...props.label }
-  return (
+
+  return props.formReadOnly ? null : (
     <FieldRowWrapper key={`row-${props.fieldIndex}`} {...getRowStyle('wrapper')}>
-      {props.label && <Label {...label} />}
+      {props.label ? (
+        <Label {...label} />
+      ) : (
+        <LabelWrapper {...getLabelStyle('wrapper')}>
```

Comment:
why do we need this?

Reply:
otherwise the description is aligned left instead of next to the form field

Reply:
But maybe there is a more elegant solution. However, this one is in production and works well for Orca ;)

Reply:
can't we apply a style override to the checklist instead?

Reply:
like "margin-left: auto" on the checklist wrapper or something